frontend/src/service/requests.js

```diff
@@ -1,18 +1,18 @@
+import axios from 'axios';
 import { BASE_URL } from '../configs/environment';
 
 /* @deprecated 의존성 완전 삭제 이후 코드 삭제*/
 export const requestGetStudylog = ({ id, accessToken }) => {
-  if (accessToken) {
-    return fetch(`${BASE_URL}/studylogs/${id}`, {
-      method: 'GET',
-      headers: {
+  const headers = accessToken
+    ? {
         'Content-Type': 'application/json; charset=UTF-8',
         Authorization: `Bearer ${accessToken}`,
-      },
-    });
-  }
+      }
+    : {};
 
-  return fetch(`${BASE_URL}/studylogs/${id}`);
+  return axios.get(`${BASE_URL}/studylogs/${id}`, {
+    headers: headers,
+  });
```

Comment:
```suggestion
  return axios.get(`${BASE_URL}/studylogs/${id}`, { headers });
```
shorthand로 하면 더 간략하게 쓸 수 있겠네여 ㅎㅎ